<launch>
  <node pkg="srp_md" name="factor_graph_node" type="factor_graph_node" output="screen"/>
  <node pkg="srp_md" name="simulate_experiment" type="simulate_experiment" output="screen"/>
  <!-- <node pkg="srp_md" name="factor_graph_node_1" type="factor_graph_node" output="log">
    <remap from="/get_goal" to="/get_goal_1"/>
  </node> -->
</launch>
